src/main/java/com/monumental/services/SessionFactoryService.java

```diff
@@ -36,4 +42,18 @@ public SessionFactoryService() {
     public SessionFactory getFactory() {
         return this.factory;
     }
+
+    @Autowired
+    private MonumentListener monumentListener;
```

Comment:
Can you move the private variable declaration to the top with `private SessionFactory factory;` ?